src/line.js

```diff
@@ -1,11 +1,16 @@
+// Get the chart variable
+var Chart = require('chart.js');
+Chart = typeof(Chart) === 'function' ? Chart : window.Chart;
+var helpers = Chart.helpers;
+
 // Line Annotation implementation
 module.exports = function(Chart) {
 	var horizontalKeyword = 'horizontal';
 	var verticalKeyword = 'vertical';
 
 	var LineAnnotation = Chart.Element.extend({
 
-		draw: function(ctx) {
+		draw: function(ctx, options) {
```

Comment:
If the options were added to the `_model` property, this would probably work better since they would automatically change during animations. Or is it intended to have these not animate?